docs/dependencies.md

```diff
@@ -40,6 +41,8 @@ dependencies:
 - file:libs/myPackage.tgz
 - req:requirements.txt
 - github:myGitHub/fakeRepo
+- pyproj:
```

Comment:
this actually is failing because the trailing colon `:` is causing this entry to be parsed as a dict instead of a string.

 Need an alternative way to specify installing just the base dependencies...

Reply:
A simple option would be to add quotes around it:`- "pyproj:"` but this is kinda ugly.

Another way would be to use an "invalid" optional dependency specifier like `- pyproj:.required`, `- pyproj:_base`, or just `- pyproj:_`.

Yet another option would be to put the colon first: `- :pyproj`, but that sort of violates the `- {type}:{source}` that skelebot created.

Reply:
The last option makes sense to me. But honestly, I think they all could work. (The colon first one is a bit odd though)

Reply:
Ok made it simpler and a bit more general. Now the option is `proj:pyproject.toml` where the part after the colon is the name of the file (just like for requirements.txt). This filename is fixed right now but who know if years down the line the Python council decides to change it!
Also, skelebot will just install all packages that it finds in that file.